core/templates/dev/head/player/AnswerClassificationServiceSpec.js

```diff
@@ -34,6 +44,7 @@ describe('Answer classification service', function() {
         value: 'content'
       }],
       interaction: {
+        id: 'RuleTest',
```

Comment:
Please make sure there are tests in this spec that result in needing to call the backend classification service.